addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/DeviceInformation.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.yamahareceiver.internal.protocol;
+
+/**
+ * The device information protocol interface.
+ *
+ * @author David Graeff - Initial contribution
+ */
+
+public interface DeviceInformation extends IStateUpdateable {
```

Comment:
The interface enforces consistency across all state classes as all of them seem to require a commong trait/method of being able to `invalidate()` them :)